bezier-rs/docs/interactive-docs/src/App.vue

```diff
@@ -55,6 +62,32 @@ export default defineComponent({
 					// eslint-disable-next-line
 					callback: (): void => {},
 				},
+				{
+					name: "Bezier from points",
```

Comment:
I think you want to call this "Bezier through points" and sort it to not be at the top.

Reply:
Good catch on the name!

I kept the function near the top because it's a variant of the constructor so it felt right to keep them together. It's also the order that is used by `bezierjs`, although we do not need to necessarily do the same thing. Do you have any particular preference for the order in general?

Reply:
I assumed you just had it at the top for convenience while testing. I trust your discretion.